app/models/entities/organization.rb

```diff
@@ -0,0 +1,34 @@
+class Entities::Organization < Maestrano::Connector::Rails::Entity
+  def self.connec_entity_name
+    'Organization'
+  end
+
+  def self.external_entity_name
+    'Organization'
+  end
+
+  def self.mapper_class
+    OrganizationMapper
+  end
+
+  def self.object_name_from_connec_entity_hash(entity)
+    entity['name']
+  end
+
+  def self.object_name_from_external_entity_hash(entity)
+    entity['name']
+  end
+end
+
+class OrganizationMapper
+  extend HashMapper
+
+  map from('name'), to('name')
+  map from('code'), to('company_id')
+  map from('number_of_employees'), to('people_count')
```

Comment:
You could add mapping of 'email/address' to 'owner_id/email' and 'email/address2' to 'cc_email'